Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -964,26 +853,27 @@ static inline bool shouldAddContentKeyRecipients()
 #endif
 }
 
-void SourceBufferPrivateAVFObjC::enqueueSample(Ref<MediaSample>&& sample, TrackID trackID)
+void SourceBufferPrivateAVFObjC::enqueueSample(Ref<MediaSample>&& sample, TrackID trackId)
 {
-    if (!isEnabledVideoTrackID(trackID) && !audioRendererForTrackID(trackID))
+    auto trackIdentifier = trackIdentifierFor(trackId);
+    if (!trackIdentifier)
         return;
```

Comment:
It's kind of weird to create this local variable and only check that it's not null but not use it any further. Consider adding a `bool hasTrackIdentifierFor(TrackID) const` method.